edx_proctoring/static/proctoring/js/views/proctored_exam_view.js

```diff
@@ -71,6 +76,22 @@ var edx = edx || {};
                 "credit eligibility in this course.\n");
         },
         updateRemainingTime: function (self) {
+            self.timerTick ++;
+            if (self.timerTick % 5 == 0){
+                var url = self.model.url + '/' + self.model.get('attempt_id');
+                $.ajax(url).success(function(data) {
+                    if (data.status === 'error') {
+                        // Let the student know that his exam has failed due to an error.
+                        // This alert may or may not bring the browser window back to the
+                        // foreground (depending on browser as well as user settings)
+                        alert(gettext('Your exam has failed'));
```

Comment:
@pbaruah Please see the conversation above. To come up with a useful message I'd like a better understanding of what state the proctoring session is left in. What is the student's situation and what guidance should we give them if proctoring ends unexpectedly?